### DISCUSSION TITLE
Positioning tooltip content directly above trigger

### DISCUSSION THREAD

**Discussion by wencyen** [chakra-ui/ark#1724] [Q&A]
Hi,
I am looking for a way to `position the tooltip content over the trigger element`? Given the following example, I've tried a few configurations, but so far to no avail. Is it possible to do this in a clean way using ark's tooltip positioning api? I dug around the source code and found the [getPlacement function](https://github.com/chakra-ui/zag/blob/3c3a207f7d1240989f8b681437ca4e64582c362b/packages/utilities/popper/src/get-placement.ts#L97-L117) from `zag-js utilities`. In particular, I see that it computes the transform values for `--x` and `--y` from a package called `@floating-ui/dom`. I believe that if I can understand how these two values are computed, then I may get the answer to my original question. 

```js
computePosition(reference, floating, {
      placement,
      middleware,
      strategy,
      ...config,
    }).then((data) => {
      const x = Math.round(data.x)
      const y = Math.round(data.y)

      floating.style.setProperty("--x", `${x}px`) // ⭐️
      floating.style.setProperty("--y", `${y}px`) // ⭐️
      // ...
    })
```

```js
<Tooltip.Root positioning={positioningProp}>
  <Tooltip.Trigger>
    <UserAvatar />
  </Tooltip.Trigger>
  <Tolltip.Positioner>
    <Tooltip.Content>
      <div>
        <UserAvatar />
        <UserName />
        <UserEmail />
      </div>
    </Tooltip.Content>
  </Tolltip.Positioner>
</Tooltip.Root>
```

```js
// Expected the offset values of 0 to position the tooltip content at the top left corner of the trigger
const positioningProp = {
  offset: { mainAxis: 0, crossAxis: 0 },
  overlap: true,
}
```
![example configuration 1](https://github.com/chakra-ui/ark/assets/49729798/98044fed-d4de-422a-9811-f7199cba1769)

```js
// This pushes the tooltip content up against the left edge of the page but does not adjust the vertical position at all
const positioningProp = {
  offset: { mainAxis: -60, crossAxis: -60 },
  placement: "left-start",
}
```
![Screen Shot 2023-11-15 at 9 22 36 PM](https://github.com/chakra-ui/ark/assets/49729798/fc2389ec-cd7c-4d1a-8ff3-3d08af5b71b8)

**Comment by cschroeter** [comment]
@codepath2019 

I'm happy to help. Could you please provide an example repository (Stackblitz etc)?

**Reply by wencyen** [reply]
Hey @cschroeter, thank you for responding. I will spin one up as soon as I have a moment. I will likely be able to do so after work today, or tomorrow.

**Reply by wencyen** [reply]
Hi, cschroeter, 

Works been busy this week. I'll try to get you the repository as soon as I can. thanks!
